lib/Page.js

```diff
@@ -722,9 +722,8 @@ class Page extends EventEmitter {
       } else {
         element.value = values.shift();
       }
-
-      element.dispatchEvent(new Event('change'));
-      element.dispatchEvent(new Event('input'));
+      element.dispatchEvent(new Event('input', {'bubbles': true, 'cancelable': true}));
+      element.dispatchEvent(new Event('change', {'bubbles': true, 'cancelable': true}));
```

Comment:
both 'input' and 'change' events are not cancelable:
- https://developer.mozilla.org/en-US/docs/Web/Events/input
- https://developer.mozilla.org/en-US/docs/Web/Events/change